packages/flutter/lib/src/cupertino/action_sheet.dart

```diff
@@ -834,7 +834,7 @@ class _CupertinoAlertContentSection extends StatelessWidget {
     if (titleContentGroup.isEmpty) {
       return SingleChildScrollView(
         controller: scrollController,
-        child: Container(
+        child: const SizedBox(
```

Comment:
you can do even better than this, `SizedBox.shrink()`

Reply:
Will fix in a new PR